sections/reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews
+
+At Open Knowledge International (OKI) all code must be reviewed by a colleague before merging into the main branch of the repository (usually `master`). This short document examples how to do code reviews, and why we do it.
+
+## How
+
+The process for code reviews is straight forward as long as you are already following our coding standards for working with version control, and in particular our git flow-influenced process of branch management. The bottom line is that, before any code hits `master`, it must be reviewed by a colleague.
+
+### Submitting code for review
+
+Submitting code for review should follow these steps:
+
+1. Work out of your own fork of the main repository (usually `origin`).
+2. Always, always work from a feature or fix branch that is checked out of `master`.
+3. Each feature or fix branch should be focused on a discrete unit of work.
+4. When your unit of work is complete, submit a pull request against the `master` branch on `origin`.
+5. Wait for the CI server to run, validating your tests pass on all target environments. If you do not have tests, or have not configured CI, do that immediately before proceeding.
+6. If CI is green, ask a colleague to review your pull request (we will automate this in the near future). This preferably will be a colleague who is not working on your project.
+7. Address any questions from the code review. Sometimes this will involving refactoring, other times it will just mean answering questions.
+8. When the reviewer indicates that the pull request is ready to merge, you may merge into `master`.
```

Comment:
As long as the reviewer indicates that the pull request is ready, I don't think it matters who "clicks the button". The reason that I do not expect the reviewer to actually merge is because the reviewer is not necessarily working on the codebase and so perhaps in these circumstances it is better to let the developer chose the time for merge.

We can change it to ask reviewer to always merge, on the assumption that the developer is expecting this to happen when the code is approved.